templates/angular/igx-ts/autocomplete/autocomplete-extended/index.ts

```diff
@@ -6,7 +6,7 @@ class IgxAutocompleteTemplate extends IgniteUIForAngularTemplate {
 		this.components = ["Autocomplete"];
 		this.controlGroup = "Data Entry & Display";
 		this.listInComponentTemplates = true;
-		this.id = "autocomplete";
+		this.id = "custom-autocomplete";
```

Comment:
Look like you accidentally switched the names of the autocomplete templates 🔼 (this one is the *extended* file).